addons/website_blog/views/website_blog_templates.xml

```diff
@@ -380,20 +386,20 @@
 <template id="index_right" name="Right Column"
         inherit_id="website_blog.blog_post_short">
     <xpath expr="//div[@id='main_column']" position="after">
-        <div class="col-xl-3 offset-xl-1 col-md-4" id="blog_right_column"/>
+        <div class="col-lg-3" id="blog_right_column"/>
     </xpath>
 </template>
 
 <!-- Option:Right Column: tags -->
 
 <template id="tag_category" name="Tags List">
-    <ul class="nav nav-pills ml16">
+    <ul class="nav nav-pills">
         <t t-if="tags">
             <div t-if='not hide_title' class='text-muted mt8'><i class='fa fa-tag' role="img" aria-label="Tag category" title="Tag category"/> <t t-esc='categ_title' /></div>
             <t t-foreach="tags" t-as="tag">
                 <t t-if="tag.post_ids">
-                    <li class="nav-item">
-                        <a t-attf-href="#{blog_url(tag=tags_list(active_tag_ids, tag.id))}" t-attf-class="nav-link#{tag.id in active_tag_ids and ' active' or ''}">
+                    <li class="nav-item mr-2">
+                        <a t-attf-href="#{blog_url(tag=tags_list(active_tag_ids, tag.id))}" t-attf-class="badge nav-link#{tag.id in active_tag_ids and ' active' or ''}">
```

Comment:
@kea14 General idea with bootstrap is composition: a component can contain others, but a component cannot be another. So a nav-link can contain a badge but not be a badge ;) If the result is good, this is just by pure luck ;) Then we switch to bootstrap 5... and the luck is gone... :)